VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -101,7 +100,7 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseComponent implemen
                     if (this.conference.audio_recording_required) {
                         this.initAudioRecordingInterval();
                     }
-                    this.isIMEnabled = this.defineIsIMEnabled();
+                    this.defineIsIMEnabled();
```

Comment:
this line does nothing, its returning a boolean, not setting anything and not being used..
please could we remove it

Reply:
I didn't notice this. Done :)